src/IssueViz.Security/Hotspots/LocalHotspotsStore.cs

```diff
@@ -214,8 +215,8 @@ private LocalHotspot MatchAndConvert(IAnalysisIssueVisualization visualization)
             return new LocalHotspot(visualization, hotspotReviewPriorityProvider.GetPriority(visualization.RuleId) ?? DefaultPriority);
         }
 
-        private IEnumerable<LocalHotspot> FlattenMapping() => 
-            fileToHotspotsMapping.SelectMany(kvp => kvp.Value);
+        private IEnumerable<LocalHotspot> FlattenMapping() =>
```

Comment:
Probably worth renaming the method to make it clear it is doing more than just flattening
e.g. `FlattenAndFilterMapping`, or even `GetOpenHotspots` / `GetUnreviewedHotspots`.

Reply:
+ 1

Reply:
I think Open makes more sense because we're getting the ones reviewed and acknowledged.